src/js/components/FormField/FormField.js

```diff
@@ -190,14 +193,38 @@ const FormField = forwardRef(
       pad,
       required,
       style,
-      validate,
+      validate: validateProp,
       validateOn,
       ...rest
     },
     ref,
   ) => {
+    const validate = isObject(validateProp) ? {...validateProp} : validateProp;
     const theme = useContext(ThemeContext) || defaultProps.theme;
     const formContext = useContext(FormContext);
+
+    if (isObject(validate) && 'max' in validate && !('threshold' in validate))
+      validate.threshold = defaultThreshold;
+
+    const isMaxAndThresholdValidation =
+      isObject(validate) && 'max' in validate && 'threshold' in validate;
+
+    const getMaxAndThresholdValidation = (value) => {
+      const { max, threshold } = validate;
+
+      return value.length / max > (threshold ?? defaultThreshold)
+        ? {
+            status: max - value.length >= 0 ? 'info' : 'error',
+            message:
+              max - value.length >= 0
+                ? `${max - value.length} characters left`
+                : `${value.length - max} character${
+                    value.length - max > 1 ? 's' : ''
+                  } over limit`,
```

Comment:
Can we use a `messages` prop for this part? That way the string can be adjusted for different languages.

For an example of this you can look at this PR: https://github.com/grommet/grommet/pull/6656/files. More specifically `languages/default.json` and `button,js` lines 232-235.

Thank you! And I apologize for not catching this in my previous reviews

Reply:
@jcfilben @taysea  Done!